doc/sphinx/inter_non-bonded.rst

```diff
@@ -559,60 +559,78 @@ DPD interaction
     Feature ``DPD`` required.
 
 This is a special interaction that is to be used in conjunction with the
-`Dissipative Particle Dynamics (DPD)` thermostat, for a general description
-of the algorithm see there. The parameters can be set via::
+:ref:`Dissipative Particle Dynamics (DPD)` thermostat.
+The parameters can be set via::
 
     system.non_bonded_inter[type1, type2].dpd.set_params(**kwargs)
 
-This command defines a velocity-dependent interaction
-between particles of the types ``type1`` and ``type2``. For a description of the input arguments
-see :class:`espressomd.interactions.DPDInteraction`. The interaction
-only has an effect if the DPD thermostat activated, and acts according to the
-temperature of the thermostat.
+This command defines an interaction between particles of the types ``type1`` and ``type2``
+that contains velocity-dependent friction and noise according 
+to a temperature set by :py:attr:`espressomd.thermostat.Thermostat.set_dpd()`
+. The parameters are
 
-The interaction consists of a dissipative force :math:`\vec{F}_{ij}^{D}` and
-a random force :math:`\vec{F}_{ij}^R`, and is decomposed into a component
-parallel and perpendicular to the distance vector of the particle pair :math:`\vec{F}_{ij}`.
-The parameters for the two parts can be chosen independently.
-The force contributions of the parallel part are
+* ``gamma``
```

Comment:
where is the temperature parameter?

Reply:
The temperature is set by the dpd thermostat (Thermostat.set_dpd()) for all particles. The friction parameters etc are set by the dpd interaction

Reply:
ah it's maybe confusing because you mention the thermostat and directly after that you continue with "The parameters are"

Reply:
I will clarify that